libr/anal/types.c

```diff
@@ -21,7 +21,7 @@ R_API int r_anal_type_set(RAnal *anal, ut64 at, const char *field, ut64 val) {
 	return false;
 }
 
-R_API void r_anal_type_del(RAnal *anal, const char *name) {
+R_API void r_anal_type_del (RAnal *anal, const char *name) {
```

Comment:
function signatures doesnt have space before (. the reason is because this way they can be easily grepped